sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentHealthcareResult.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// DocumentHealthcareEntities.
+    /// </summary>
+    public partial class DocumentHealthcareResult
+    {
+        internal DocumentHealthcareResult(DocumentHealthcareEntitiesInternal documentHealthcareEntities)
```

Comment:
when this constructor is called, what is the value for `TextAnalyticsError` ? will it be empty collection or null. -> make sure it is empty collection